stdlib/2/subprocess.pyi

```diff
@@ -4,7 +4,7 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Union, Optional, List, Text
 
-_FILE = Union[int, IO[Any]]
+_FILE = Optional[Union[int, IO[Any]]]
```

Comment:
No, just style. Optional[T] is just a shorthand for Union[None, T], so I find it clearer to collapse it into a single union. There are also a lot more instances of `Union[None` (39) than `Optional[Union]` (19) in existing typeshed stubs.

Not the most important thing though.